website/docs/special-topics/asyncworker.md

```diff
@@ -5,69 +5,69 @@ prev: special-topics.object-function-refs
 
-[**SimpleAsyncWorker.h**](https://github.com/nodejs/node-addon-examples/blob/master/napi-asyncworker-example/node-addon-api/src/SimpleAsyncWorker.h)
+[**SimpleAsyncWorker.h**](https://github.com/nodejs/node-addon-examples/blob/main/napi-asyncworker-example/node-addon-api/src/SimpleAsyncWorker.h)
 
 `embed:napi-asyncworker-example/node-addon-api/src/SimpleAsyncWorker.h`
 
-This code declares a constructor that takes as an argument the length of time (in seconds) the `SimpleAsyncWorker` is to run. A private data member is declared to hold this value. 
+This code declares a constructor that takes as an argument the length of time (in seconds) the `SimpleAsyncWorker` is to run. A private data member is declared to hold this value.
 
-The header also declares two methods, `Execute` and `OnOK`, which override methods declared by `AsyncWorker`, and are described in more detail below. 
+The header also declares two methods, `Execute` and `OnOK`, which override methods declared by `AsyncWorker`, and are described in more detail below.
 
 And here's the C++ implementation file for `SimpleAsyncWorker`:
 
-[**SimpleAsyncWorker.cc**](https://github.com/nodejs/node-addon-examples/blob/master/napi-asyncworker-example/node-addon-api/src/SimpleAsyncWorker.cc)
+[**SimpleAsyncWorker.cc**](https://github.com/nodejs/node-addon-examples/blob/main/napi-asyncworker-example/node-addon-api/src/SimpleAsyncWorker.cc)
 
 `embed:napi-asyncworker-example/node-addon-api/src/SimpleAsyncWorker.cc`
 
-The constructor takes two arguments. `callback` is the JavaScript function that gets called when the `Execute` method returns. `callback` gets called whether there was an error or not. The second constructor argument, `runTime`, is an integer value indicating how long (in seconds) the worker is to run. 
+The constructor takes two arguments. `callback` is the JavaScript function that gets called when the `Execute` method returns. `callback` gets called whether there was an error or not. The second constructor argument, `runTime`, is an integer value indicating how long (in seconds) the worker is to run.
+
+Node will run the code of the `Execute` method in a thread separate from the thread running Node's main event loop. The `Execute` method _has no access_ to any part of the Node-API environment. For this reason, the `runTime` input value was stored by the constructor as a private data member.
 
-Node will run the code of the `Execute` method in a thread separate from the thread running Node's main event loop. The `Execute` method _has no access_ to any part of the N-API environment. For this reason, the `runTime` input value was stored by the constructor as a private data member. 
+In this implementation, the `Execute` method simply waits the number of seconds specified earlier by `runTime`. This is where the long running code goes in a real implementation. To demonstrate how error handling works, this `Execute` method declares an error when requested to run 4 seconds.
```

Comment:
I apologize for the mess here. I accidentally engaged my Markdown linter that removed spaces at the ends of the lines which has completely confused the comparator.